### DISCUSSION TITLE
how to change vercel production branch

### DISCUSSION THREAD

**Discussion by sujitkhandagale** [vercel/community#4430] [Help] [answered]
### Summary

i tried to change in git settings to change manually, but it wont reflect in depolyment.

### Steps to Reproduce

n/a

**Comment by amyegan** [comment, marked as answer]
By default, the `main`, `master`, or default branch of the repo is used as the `production` branch. You can [customize the production branch](https://vercel.com/docs/deployments/git#customizing-the-production-branch) on the `Git` page in `Project Settings`. After saving, the change will take effect the next time a new commit is pushed that generates a new deployment.

https://vercel.com/docs/deployments/git#production-branch
